eng/mgmt/automation/init.sh

```diff
@@ -1,9 +1,14 @@
 #!/bin/sh
 
-sudo apt-get install -y --upgrade python3-pip python3-setuptools
-pip3 install --upgrade wheel
-pip3 install --upgrade PyYAML requests
+echo Install Python
+sudo apt-get install -y --upgrade python3-pip python3-setuptools 2>&1
+pip3 install --upgrade wheel 2>&1
+pip3 install --upgrade PyYAML requests 2>&1
```

Comment:
`2>&1` is not related to tsp-client
It is an improvement to SDK automation output, to avoid irrelevant info get printed to error message.